WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -13,9 +14,18 @@ final class OrdersTests: XCTestCase {
         try LoginFlow.logInWithWPcom()
     }
 
-    // TODO: Write real test, this is a placeholder for now
-    func testGoToOrdersScreen() throws {
-        try TabNavComponent().goToOrdersScreen()
-        XCTAssert(try OrdersScreen().isLoaded)
+    func testOrdersScreenLoads() throws {
+            let orders = try GetMocks.readOrdersData()
+            print(orders) // just for human validation
+
+            _ = try TabNavComponent()
```

Comment:
Don't think we need to assign this to _ anymore. If it's still throwing an error then maybe a missing `@discardableResult` tag on one of the functions.

```suggestion
           try TabNavComponent()
```

Reply:
Ah, yep! We needed `@discardableResult` on `verifyOrdersScreenLoaded`.